map/set-initial-map-location/README.md

```diff
@@ -14,18 +14,20 @@ When the map loads, note the specific location and scale of the initial map view
 
 ## How it works
 
-1. Create a `Map`, specifying a basemap type, latitude and longitude in WGS84, and a level of detail.
+1. Create a `Map`, specifying a basemap style.
```

Comment:
This should either say `ArcGISMap` or 'map' without backticks. Easy to forget that `Map` == Key-value-Pair!

```suggestion
1. Create a `ArcGISMap`, specifying a basemap style.
```

Might be a good thing to create an issue to search for `Map` in backticks in all readme files and get rid of those relics once and for all ;)

Reply:
Ah cool, I will make a note 👍

Reply:
Just did a search and it's actually only one other sample - so I'll update that in the relevant PR when I get back to it!